tasks/23_seek-destroy/seek-destroy.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable prefer-rest-params */
```

Comment:
I think it would be okay to use `...rest` in future, as arguments here mostly for learning purpose.